src/features/addUncheckAllToReplyToModal.ts

```diff
@@ -0,0 +1,66 @@
+import './addUncheckAllToReplyToModal.css';
+
+import {makeBTDModule} from '../types/btdCommonTypes';
+
+enum LABELS {
+  SELECT_ALL = 'Select All',
+  UNSELECT_ALL = 'Unselect All',
+}
+
+export const addUncheckAllToReplyToModal = makeBTDModule(({TD}) => {
+  const jsModalsContainer = document.querySelector('.js-modals-container') as HTMLDivElement;
```

Comment:
`document.querySelector` is generic so you should do this instead. Your type casting here removes the `| undefined` that is usually there so you could end up in a situation where `.js-modals-container` is not defined yet TypeScript won't bat an eye.

```suggestion
  const jsModalsContainer = document.querySelector<HTMLDivElement>('.js-modals-container');
```